gpkit/constraints/model.py

```diff
@@ -134,14 +134,17 @@ def _add_modelname_tovars(self, name, num):
             self.subinplace(add_model_subs)
 
     # pylint: disable=too-many-locals
-    def debug(self, verbosity=1, **solveargs):
+    def debug(self, solver=None, verbosity=1, **solveargs):
         "Attempts to diagnose infeasible models."
         from .relax import ConstantsRelaxed, ConstraintsRelaxed
         from .bounded import Bounded
 
         sol = None
         relaxedconsts = False
 
+        solveargs["solver"] = solver
```

Comment:
I think this should only happen if `solver is not None` -- in case `solver` is specified in `solveargs`, I wouldn't want that to be overwritten by None`.